js/packages/teams-ai/src/Application.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable security/detect-object-injection */
```

Comment:
Remove

Reply:
this lint issue shows up if i try to do `state.conversation.value[KEY]` where `KEY` is a variable. Apparently this pattern can cause injection attacks. It doesn't apply to our scenario since `KEY` is a constant string.

Reply:
Do it per line, not per file.

Reply:
There's a bunch of files with the at the top of the file. Can we just tweak the linter to ignore this warning?

Reply:
If that's true, I failed to catch it on in-going and they should be removed and added inline as well. If you want to have the rule removed, please set up a meeting with the team to discuss.

Reply:
Due to time constraints, I'll stick to making it inline. For other places where it's done on top of the file, I'll create a new ticket.

Reply:
Okay to file the issue and keep these files included in that list instead of adding inline for files in this PR.

Reply:
Created issue: [#801](https://github.com/microsoft/teams-ai/issues/801). We don't need to keep a track of the files. We can just do a global CTRL+F to figure out where all it is.